packages/zosfiles/__tests__/__system__/cli/hMigrate/ds/cli.files.HMigrate.ds.system.test.ts

```diff
@@ -44,6 +45,7 @@ describe("Migrate Dataset", () => {
     user = defaultSystem.zosmf.user.trim().toUpperCase();
     dataSetName1 = `${user}.SDATAC.SET`;
     dataSetName2 = `${user}.PDATAC.SET`;
+    dataSetName3 = `${user}.FAIL.SET`;
```

Comment:
I wonder why we didn't make the names the same 😋 
JK, no need to change the LLQ to MIGR 😸

Reply:
I suppose the `.MIGR.` would make a lot more sense, same for recall, `.REC.`

Reply:
@zFernand0 changed this in the wrong PR 😅 changed it in #695.